okio/src/main/java/okio/ByteString.java

```diff
@@ -101,6 +101,24 @@ public String base64() {
     return Base64.encode(data);
   }
 
+  /** Returns the MD5 hash of this byte string. */
+  public String md5() {
```

Comment:
Hmmmm . . . I was thinking this should return a ByteString, and make it the caller's responsibility to call `hex()`.

(On the Cash server, we store binary hashes directly in our database, which reduces the storage requirement by 50%).